src/datatable.js

```diff
@@ -263,6 +270,25 @@ class DataTable {
     translate(str, args) {
         return this.translationManager.translate(str, args);
     }
+    setupSaveSorting() {
+        // add options in default headerdropdown
+        let action = {
+            label: this.translate('Save Sorting'),
+            action: function (column) {
+                this.saveSorting(column.colIndex, column.sotOrder);
```

Comment:
```suggestion
                this.saveSorting(column.colIndex, column.sortOrder);
```

@sokumon Spelling mistake?